go-controller/pkg/config/config.go

```diff
@@ -757,7 +757,13 @@ func (a *OvnDBAuth) SetDBAuth() error {
 			// Must happen *before* setting the "ovn-remote"
 			// external-id.
 			if a.ctlCmd == "ovn-sbctl" {
-				out, err := runOVSVsctl("set-ssl", a.PrivKey, a.Cert, a.CACert)
+				out, err := runOVSVsctl("del-ssl")
+				if err != nil {
+					return fmt.Errorf("error deleting ovs-vsctl SSL "+
```

Comment:
How well does this work in previous versions?

Reply:
It works fine with earlier versions.